app/src/examples/ArticleProgressExample.tsx

```diff
@@ -13,6 +13,10 @@ export default function ArticleProgressExample() {
   const scrollHandler = useScrollViewOffset(scrollViewRef);
 
   const progressBarAnimatedStyle = useAnimatedStyle(() => {
+    if (!textRef.current || !scrollViewRef.current) {
```

Comment:
Looks like we pass React refs to a worklet, do we really want to do it?

Reply:
Why not? We were already using ref inside worklet:
```javascript
const measuredText = measure(textRef);
```
And this seems to be the best way to check if component is already rendered...

Reply:
@Latropos Turns out it doesn't work as expected, animated refs don't have `.current` property when used on UI runtime. We'll probably need to revert this PR and find another fix. cc @jfedak whom I've complained to about ArticleProgressExample not working on macOS